src/content/1.7/development/components/global-components.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Global JavaScript components
+weight: 70
+---
+
+# Global JavaScript components?
+
+It's a [new system](https://github.com/PrestaShop/PrestaShop/blob/develop/admin-dev/themes/new-theme/js/app/utils/init-components.js) mainly created for module developers, which allow you to use components we already use in the core without importing it. [Here is the ADR](https://github.com/PrestaShop/ADR/blob/master/0009-expose-js-components-using-window-variable.md) where we decided what we should do to avoid this problem.
+
+You are now able to use components we expose to the `window.prestashop.component` object without importing anything.
+
+This object contains two attributes:
+
+- `component` which contain some components you can instantiate;
+- `instance` which contains every instances of initiated components.
+
+It allows module developers to avoid importing path such as `../../../../admin-dev/themes/new-module/js/components/translatable-field` because it was making hard to add a CI in order to build the module.
+
+## How to use?
+
+There is an event sent after the execution of `initPrestashopComponents` which you can stick on: `PSComponentsInitiated`.
+
+If you have the `events` package, you'll be able to catch the event like this:
```

Comment:
@NeOMakinG just for confirmation, `EventEmitter` is available globally? we can use `EventEmitter.on` or do we need to import `events`?

Reply:
Depends on which version you are, it's available globally only after 1.7.8

Reply:
@NeOMakinG Could you add anot about that. And it's ok for me.